apps/auth/config.py

```
@@ -27,3 +27,37 @@
+    print("\033[33m[WARN]\033[0m ISSECURE not set, using default value of False")
+try:
+    ISSECURE = bool(os.getenv("ISSECURE", False))
+except ValueError:
+    raise ValueError(
+        "\033[31m[ERROR]\033[0m \033[107mISSECURE must be a boolean value, please set it in the .env file\033[0m"
+    )
+if "google" in os.getenv("SOCIAL_LOGINS"):
+    if not os.getenv("GOOGLE_CLIENT_ID"):
+        raise ValueError(
+            "\033[31m[ERROR]\033[0m \033[107mGOOGLE_CLIENT_ID not set, please set it in the .env file\033[0m"
+        )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)